plugins/rbac-backend/openapi.yaml

```
@@ -0,0 +1,636 @@
+                          name
+                      policy:
+                        type: string
+                        description: >-
+                          Policy action for the permission: create, read,
+                          update, delete, use
+                      effect:
+                        type: string
+                        description: allow or deny
+      responses:
+        '200':
+          description: Request was successful.
+        '400':
+          description: Input Error
+        '403':
+          description: Refusal to authorize
+    delete:
+      description: >-
+        Deletes a permission policy or a group of permission policies of a
+        specified entity.
+      parameters:
+        - $ref: '#/components/parameters/nameParam'
+        - $ref: '#/components/parameters/namespaceParam'
+        - $ref: '#/components/parameters/kindParam'
+        - name: permission
+          in: query
+          description: Permission from a specific plugin, Resource type or name
+          required: false
+          schema:
+            type: string
+        - name: policy
+          in: query
+          description: 'Policy action for the permission: create, read, update, delete, use'
+          required: false
+          schema:
+            type: string
+        - name: effect
+          in: query
+          description: allow or deny
+          required: false
+          schema:
+            type: string
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                '$ref': '#/components/schemas/PermissionPolicy'
+      responses:
+        '204':
+          description: ok
+        '400':
+          description: Input Error
+        '403':
+          description: Refusal to authorize
+  /api/permission/plugins/policies:
+    get:
+      description: >-
+        Lists all plugin permission policies from plugins installed in your
+        Backstage instance.
+      responses:
+        '200':
+          description: Request was successful
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  pluginId:
+                    type: string
+                  policies:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        isResourced:
+                          type: boolean
+                        permission:
+                          type: string
+                          description: Permission from a specific plugin.
+                        policy:
+                          type: string
+                          description: >-
+                            Policy action for the permission: create, read,
+                            update, delete, use.
```

We have API changes with the latest main branch:
```suggestion
              schema:
                type: array
                items: 
                  type: object
                  properties:
                    pluginId:
                      type: string
                    policies:
                      type: array
                      items:
                        type: object
                        properties:
                          name:
                            type: string
                            description: Permission from a specific plugin.
                          resourceType:
                            type: string
                            description: Resource type.
                          policy:
                            type: string
                            description: >-
                              Policy action for the permission: create, read,
                              update, delete, use.
                        required: [name, policy]
```